app/scripts/modules/core/pipeline/config/triggers/webhook/webhookTrigger.module.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.trigger.webhook', [
+    require('../../../../../core/config/settings.js'),
+  ])
+  .config(function (pipelineConfigProvider) {
+    pipelineConfigProvider.registerTrigger({
+      label: 'Webhook',
+      description: 'Executes the pipeline on a Webhook POST',
+      key: 'webhook',
+      controller: 'WebhookTriggerCtrl as ctrl',
```

Comment:
This should just be `controller: 'WebhookTriggerCtrl',` - the `as ctrl` part conflicts with the next line (`controllerAs: 'vm'`). It looks like Angular is discarding the `as ctrl` bit in favor of the `vm` piece, but let's avoid the confusion altogether.